build.gradle

```
@@ -8,6 +8,8 @@ plugins {
   id 'com.dorongold.task-tree' version '2.1.0' apply false
   id 'com.github.johnrengelman.shadow' version '7.1.2' apply false
   id 'com.github.sherter.google-java-format' version '0.9' apply false
+
+  id "org.sonarqube" version "3.3"
```

Super-nit: just a style thing. I prefer single quotes that don't require evaluation.
```suggestion
  id 'org.sonarqube' version '3.3'
```